selenium/che-selenium-test/src/test/java/org/eclipse/che/selenium/dashboard/organization/CreateOrganizationTest.java

```diff
@@ -0,0 +1,99 @@
+/*
+ * Copyright (c) 2012-2017 Red Hat, Inc.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ *
+ * Contributors:
+ *   Red Hat, Inc. - initial API and implementation
+ */
+package org.eclipse.che.selenium.dashboard.organization;
+
+import static org.eclipse.che.commons.lang.NameGenerator.generate;
+import static org.eclipse.che.selenium.pageobject.dashboard.NavigationBar.MenuItem.ORGANIZATIONS;
+import static org.eclipse.che.selenium.pageobject.dashboard.organization.OrganizationListPage.OrganizationListHeader.NAME;
+import static org.testng.Assert.assertEquals;
+import static org.testng.Assert.assertTrue;
+
+import com.google.inject.Inject;
+import com.google.inject.name.Named;
+import org.eclipse.che.multiuser.organization.shared.dto.OrganizationDto;
+import org.eclipse.che.selenium.core.client.TestOrganizationServiceClient;
+import org.eclipse.che.selenium.core.user.AdminTestUser;
+import org.eclipse.che.selenium.pageobject.dashboard.Dashboard;
+import org.eclipse.che.selenium.pageobject.dashboard.NavigationBar;
+import org.eclipse.che.selenium.pageobject.dashboard.organization.AddOrganization;
+import org.eclipse.che.selenium.pageobject.dashboard.organization.OrganizationListPage;
+import org.eclipse.che.selenium.pageobject.dashboard.organization.OrganizationPage;
+import org.testng.annotations.AfterClass;
+import org.testng.annotations.BeforeClass;
+import org.testng.annotations.Test;
+
+/**
+ * Test validates organization creation
+ *
+ * @author Ann Shumilova
+ */
+public class CreateOrganizationTest {
+  private static final String ORGANIZATION_NAME = generate("organization", 5);
+  private static final String SUB_ORGANIZATION_NAME = generate("organization", 5);
+
+  @Inject
+  @Named("admin")
+  private TestOrganizationServiceClient testOrganizationServiceClient;
+
+  @Inject private OrganizationListPage organizationListPage;
+  @Inject private OrganizationPage organizationPage;
+  @Inject private AddOrganization addOrganization;
+  @Inject private NavigationBar navigationBar;
+  @Inject private AdminTestUser adminTestUser;
+  @Inject private Dashboard dashboard;
+
+  @BeforeClass
+  public void setUp() throws Exception {
+    dashboard.open(adminTestUser.getName(), adminTestUser.getPassword());
+  }
+
+  @AfterClass
+  public void tearDown() throws Exception {
+    for (OrganizationDto organization : testOrganizationServiceClient.getAll())
+      testOrganizationServiceClient.deleteById(organization.getId());
```

Comment:
Such code can affect organizations which are created in parallel threads of test execution.